test/unit/client-side-encryption/state_machine.test.ts

```diff
@@ -83,6 +83,7 @@ describe('StateMachine', function () {
   });
 
   describe('kmsRequest', function () {
+    let sandbox: sinon.SinonSandbox;
```

Comment:
I refactored `this.sinon.stub` -> `sandbox.stub` while debugging, and although that didn't fix the tests, I left it in because we generally use scoped variables instead of binding variables to `this` of the executing test.